doc/changes/changes_3.0.1.md

```diff
@@ -0,0 +1,38 @@
+# Virtual Schema for Amazon DynamoDB 3.0.1, released 2022-06-24
```

Comment:
Please don't forget to update the date

Reply:
Release date is updated, thanks for reminder.